ga4gh/datamodel/genotype_phenotype.py

```diff
@@ -0,0 +1,624 @@
+"""
+Module responsible for translating g2p data into GA4GH native
+objects.
+"""
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import collections
+import rdflib
+
+import ga4gh.datamodel as datamodel
+import ga4gh.exceptions as exceptions
+import ga4gh.protocol as protocol
+
+# annotation keys
+TYPE = 'http://www.w3.org/1999/02/22-rdf-syntax-ns#type'
+LABEL = 'http://www.w3.org/2000/01/rdf-schema#label'
+HAS_QUALITY = 'http://purl.obolibrary.org/obo/BFO_0000159'
+
+
+class AbstractPhenotypeAssociationSet(datamodel.DatamodelObject):
+    compoundIdClass = datamodel.PhenotypeAssociationSetCompoundId
+
+    def __init__(self, parentContainer, localId):
+        super(AbstractPhenotypeAssociationSet, self).__init__(
+            parentContainer, localId)
+
+    def toProtocolElement(self):
+        pas = protocol.PhenotypeAssociationSet()
+        pas.name = self.getLocalId()
+        pas.id = self.getId()
+        pas.dataset_id = self.getParentContainer().getId()
+        return pas
+
+
+class SimulatedPhenotypeAssociationSet(AbstractPhenotypeAssociationSet):
+    def __init__(self, parentContainer, localId, randomSeed):
+        super(SimulatedPhenotypeAssociationSet, self).__init__(
+            parentContainer, localId)
+
+    # TODO this doesn't make much sense
+    def getAssociations(
+            self, request=None, featureSets=[]):
```

Comment:
It's being addressed in your PR which hasn't been merged into the g2p branch yet